### DISCUSSION TITLE
Compiling tractus edc worked yesterday, fails today

### DISCUSSION THREAD

**Discussion by lucian-torje-siemens** [eclipse-edc/Connector#3078] [General]
Hello,
I successfully compiled yesterday tractusX 0.3.3 which uses edc and today it fails due to the Java 11 --> Java 17 upgrade done in EDC see https://github.com/eclipse-edc/Connector/commit/25a46165fb010cc9a4990180da5c926bf29ffab3.
Maybe bumping version for edc build on such changes could help.

![image](https://github.com/eclipse-edc/Connector/assets/102717569/eabbd689-ae70-4d06-8043-9d8d77edfbbf)

**Comment by juliapampus** [comment]
Is this something related to the EDC project or Tractus-X specific? If the latter, I would ask you to move the discussion to the appropriate repository.

**Comment by ndr-brt** [comment]
definitely wrong repository
